CONTRIBUTING.md

```diff
@@ -1,78 +0,0 @@
-### Introduction
```

Comment:
GitHub links to this file when opening a new PR

Reply:
Changed the urls to relative. Is that what you mean?

Reply:
No, you shouldn't remove this file.

If you open a PR the first time to a repo, you will see
```
It looks like this is your first time opening a pull request in this project!

Be sure to review the contributing guidelines.
```
contributing guidelines has link to CONTRIBUTING.md in root directory

Reply:
Now it makes sense. You're talking about https://github.com/xbmc/xbmc/community links.
I want to declutter root dir and, to the best of my knowledge, CONTRIBUTING.md and PULL_REQUEST_TEMPLATE.md can be in project root dir, inside `docs/` or `.github/` and are not case sensitive. What remains to be seen is if GitHub picks it up automatically (I think so).